backend/audit/test_views.py

```diff
@@ -1656,3 +1813,316 @@ def test_valid_file_upload_for_notes_to_sefa(self, mock_scan_file):
             submission_events[event_count - 1].event,
             SubmissionEvent.EventType.NOTES_TO_SEFA_UPDATED,
         )
+
+
+class EditSubmissionTest(TestCase):
```

Comment:
Should these be in the same class as `EditSubmissionViewTests` above?

Reply:
Moved and merged into the other class. Originally they were testing one with a logged in session and one without, but I believe I know handled both cases.